src/test/java/entities/TagTest.java

```diff
@@ -11,15 +11,15 @@
 import static org.junit.Assert.*;
```

Comment:
Is it possible to uncommit this file? We already deleted it so I don't know what's going to happen once we merge this PR.